src/state/migrations/1.test.ts

```diff
@@ -0,0 +1,50 @@
+import { RouterPreference } from 'state/routing/types'
+import { UserState } from 'state/user/reducer'
+import { SlippageTolerance } from 'state/user/types'
+
+import { migration1, PersistAppStateV1 } from './1'
+
+const previousState: PersistAppStateV1 = {
+  user: {
+    userLocale: null,
+    userRouterPreference: RouterPreference.API,
+    userHideClosedPositions: false,
+    userSlippageTolerance: SlippageTolerance.Auto,
+    userSlippageToleranceHasBeenMigratedToAuto: true,
+    userDeadline: 1800,
+    tokens: {},
+    pairs: {},
+    timestamp: Date.now(),
+    hideBaseWalletBanner: false,
+  },
+  _persist: {
+    version: 0,
```

Comment:
nit: would this version be updated after running `migration1`? Can you check if it's updated.